Tracking Plan spreadsheet update (#991)
rudderlabs/rudderstack-docs: docs/features/data-governance/tracking-plans/tracking-plan-spreadsheet.mdx

By default, the Tracking Plan configuration is in JSON format. The [Tracking Plan spreadsheet](https://docs.google.com/spreadsheets/d/1QnCY3dbzyz2InB68W3AGIJshsuY3Zu8WA3AfMMxIzAQ/edit#gid=1810167975) is a no-code, spreadsheet representation of your Tracking Plan configuration. It lets you easily create or use an existing Tracking Plan.

<div class="successBlock">

> To make a copy of your own Tracking Plan spreadsheet, click the above <a href="https://docs.google.com/spreadsheets/d/1QnCY3dbzyz2InB68W3AGIJshsuY3Zu8WA3AfMMxIzAQ/edit#gid=1810167975">link</a> and click <strong>File</strong> > <strong>Make a copy</strong> in the Google Sheets menu.

Reviewer: ```suggestion
To use the Tracking Plan spreadsheet, go to the <a href="https://docs.google.com/spreadsheets/d/1QnCY3dbzyz2InB68W3AGIJshsuY3Zu8WA3AfMMxIzAQ/edit#gid=1810167975">spreadsheet</a> and click <strong>File</strong> > <strong>Make a copy</strong> in the Google Sheets menu.
```
